Dockerfile

```diff
@@ -52,13 +52,14 @@ RUN npm install --global \
     xo-server-load-balancer \
     xo-import-servers-csv
 
+RUN rc-update add nfsmount
```

Comment:
your enabling `nfsmount` on start ... well great but we never start any rc daemon in the container, the `ENTRYPOINT` use `tini` to start the `entrypoint.sh` script, and keep running this script. So `nfsmount` should not run on start.